src/components/PopupWithForm.js

```diff
@@ -13,7 +13,7 @@ export default class PopupWithFrom extends Popup {
   }
 
   _getInputValues() {
-    const inputsObject = {}; // object for storing data
+    const inputsObject = {};
     this._inputList = this._popupForm.querySelectorAll(".modal__input");
```

Comment:
**Could be improved**
All elements that are found every time you call a method should be defined as **class fields** in the constructor, `_setEventListeners` (or in `generateCard`) only **1 time** not to waste resources on searching them again and again.

`Class fields` (the so called **class variables** with `this`) are accessible inside any method of the class. You shouldn’t pass **class fields** into the call of any method because you can use class fields inside all methods.
There can be `this._inputs`, `this._submitButton`, `this._cardImage`, `this._form` and so on.